src/tmb/font.cpp

```diff
@@ -63,7 +63,20 @@ void fontSetCharWidth(int font, int char_width)
     fontInfo[font].char_width = char_width;
 }
 
-INCLUDE_ASM("asm/nonmatchings/tmb/font", fontSetDefaultSize__Fi);
+void fontSetDefaultSize(int font)
+{
+    switch (font) {
+    case 0:
+        fontSetSize(font, 16);
+        fontSetSpacing(font, 2);
+        break;
+
+    case 1:
+        fontSetSize(font, 12);
+        fontSetSpacing(font, 1);
+        break;
+    }
+}
```

Comment:
It looks like I added `-fno-exceptions` to the C++ flags thinking it would cover this, but it actually also needs to be added to the C compiler flags as well.

Can you edit `tools/build_system/env.py` line 182 and add `"-fno-exceptions"` to the `c_flags` argument of `EnvironmentToolchain()`?